theme/templates/resource-landing-page/coverage.html

```diff
@@ -290,9 +302,17 @@ <h3>Coverage</h3>
                      </button>
                 {% endif %}
             </div>
-
-            <div class="col-xs-12"><h4>Temporal:</h4></div>
-
+            <div class="col-xs-12" style="margin-top: 20px;">
+                <div class="row">
+                    <div class="col-xs-1">
+                        <h4>Temporal:</h4></div>
+                    <div class="col-xs-11">
```

Comment:
@pkdash, same issue here for mobile devices.

Reply:
Fixed as above.